recipes-core/busybox/busybox_1.%.bbappend

```diff
@@ -13,11 +13,11 @@ SRC_URI =+ " \
             file://login-utilities.cfg \
             file://udhcpd.wlan0.conf"
 
-SRC_URI:append:x64 += "file://enable_ar_create_fragment.cfg"
+SRC_URI:append:x64 = " file://enable_ar_create_fragment.cfg"
```

Comment:
Check this. I think we probably want `SRC_URI:x64 += ...` instead.

Reply:
From output of `bitbake -e busybox`, `SRC_URI` does contains `file://enable_ar_create_fragment.cfg`

Reply:
Yeah; I expect that. But did it contain strings other than that fragment file prior to executing the bbappend? If this logic were wrong, I would expect it to overwrite the existing `SRC_URI_append` value with the fragment file.

Reply:
Yes, the variable is same as without these changes i.e., the logic didn't change. Same for rest of the changes here.